frontend/test/metabase/scenarios/joins/reproductions/27380-dashboard-drops-joined-fields-on-zoom-in.cy.spec.js

```diff
@@ -40,10 +40,10 @@ describe.skip("issue 27380", () => {
 
     // Graph should still exist
     // Let's check only the y-axis label
-    cy.get("y-axis-label").invoke("text").should("eq", "Count");
+    cy.get(".y-axis-label").invoke("text").should("eq", "Count");
 
     cy.icon("notebook").click();
     cy.findByText("Pick a column to group by").should("not.exist");
-    cy.findByText(/Products? → Created At: Month/);
+    cy.findByText("Count by Product → Created At: Week");
```

Comment:
@nemanjaglumac not sure if another title was intended. Could you double check?

Reply:
At least currently we override `temporal-unit` from the original breakout to match the default one for the column https://github.com/metabase/metabase/blob/5e839b3556de06070e0eaf0bcfce915c37aa5f03/frontend/src/metabase-lib/queries/utils/drilldown.js#L454. I don't think we should change it here. I'd like this PR to address only foreign keys not working with `zoom in` drilldown. WDYT?

Reply:
Tbh, I don't remember.
I was following the repro steps and in the last step it says: "The group by column gets dropped". The group by column was "Product -> Created At: Month", so I guess that was my reasoning.